js/content/store.js

```diff
@@ -1385,10 +1385,17 @@ class AppPageClass extends StorePageClass {
         if (this.isDlc()) { return; }
 
         let appid = this.appid;
-        let response = await Background.action("appdetails", {"appids": appid, "filters": "support_info"});
-        if (!response || !response[appid] || !response[appid].success) { return; }
+        let cache = JSON.parse(LocalStorage.get("support_infos") || "{}");
```

Comment:
You don't need to parse that here

Reply:
You sure? Last time I tried that it gave an error.

Reply:
chrome.storage should support object values, maybe you're thinking about localStorage